airbyte-workers/src/main/java/io/airbyte/workers/DefaultReplicationWorker.java

```diff
@@ -155,28 +155,59 @@ else if (hasFailed.get()) {
         outputStatus = ReplicationStatus.COMPLETED;
       }
 
+      final StreamNameToBytesEmitted streamNameToBytesEmitted = new StreamNameToBytesEmitted();
```

Comment:
ah. see my comment in `ReplicationAttemptSummary.yaml`. the way this class is getting generated right now is a little risky. it's not much better than a generic `Object` in java. I think we can specify the types in json schema to get better type safety.